### PR TITLE
Add chicken

### PR DESCRIPTION
**Pull request by Yoshyyy**

### CODE REVIEW

File: animalfacts.py [hunk 1]
```diff
@@ -2397,6 +2406,12 @@ def animalfactsbot(reddit):
     'The oldest known wild Common Raven was at least 22 years, 7 months old. It was banded and found in Nova Scotia.'
     )
 
+REINDEER_FACTS = (
+    'Reindeers, contrary to popular belief, can not actually fly',
+    'Reindeers do not actually live in the north pole, and can be found in Lapland.',
```
Comment (mpentler) [inline animalfacts.py]: “at the North Pole” I think. I’m a stickler for grammar :)

File: animalfacts.py [hunk 2]
```diff
@@ -2397,6 +2406,12 @@ def animalfactsbot(reddit):
...
+    'Reindeers are actually bigger than most people think'
```
Comment (mpentler) [inline animalfacts.py]: Needs a comma at the end here.

File: animalfacts.py [hunk 3]
```diff
@@ -2856,7 +2871,7 @@ def animalfactsbot(reddit):
     'Vampire bats can live for up to 12 years in the wild, although captive individuals have been known to reach the age of 19.',
     'Young vampire bats feed on milk from the mother, not on blood. They cling tightly to their mothers, even in flight, until they are weaned at about 3 – 4 months.',
     'Vampire bats are not seasonal breeders and can mate all year round. They are pregnant for 3 – 4 months, which is a long gestation period compared to other small bats. The female gives birth to a single baby.',
-    'The saliva of a vampire bat can be used to prevent blood from clotting.',	
+    'The saliva of a vampire bat can be used to prevent blood from clotting.',
```
Comment (mpentler) [inline animalfacts.py]: Thanks for fixing this! :)

**Comment by bb128b** [comment]
I have fixed conflicts in the following PR (https://github.com/joel-g/animal-facts-bot/pull/366). Not sure if this was the right way to do it. This PR has been stagnant for 28 days now.

**Comment by mpentler** [comment]
I’m gonna close this then and work on yours @bb128b - thanks for your efforts!

**Pull request closed without merging**